services/services.rst

```diff
@@ -0,0 +1,209 @@
+Overview of services
+******
+
+Below are tools and resources that are used in the PFS project. 
+Many services are hosted at IPMU under `pfs.ipmu.jp <https://pfs.ipmu.jp>`_ 
```

Comment:
pfs.ipmu.jp (https)  ?
 
just following the account.rst

Reply:
added